app/src/test/java/comp3350/mealbuddy/tests/business/CalculatorTest.java

```diff
@@ -383,6 +398,53 @@ public void getLabelCalories_allEmptyMeals_returnZero() {
         Assert.assertEquals(expectedCalories, actualCalories);
     }
 
+    @Test
+    public void getExerciseCalories_oneExercise_returnCalories() {
+        int expectedCalories = 952;
+        UserInfo user = createDefaultUser();
+        Calculator calculator = new Calculator(day);
+        Exercise exercise = new Exercise("bike", 60.0, Exercise.Intensity.High);
+        int actualCalories = calculator.getExerciseCalories(exercise, user);
+
+        Assert.assertEquals(expectedCalories, actualCalories);
+
+    }
+
+    @Test
+    public void getExerciseCalories_multipleExercises_returnSummedCalories() {
+        int expectedCalories = 1713;
+        UserInfo user = createDefaultUser();
+        Calculator calculator = new Calculator(day);
+        int actualCalories = calculator.getTotalExerciseCalories(user);
+
+        Assert.assertEquals(expectedCalories, actualCalories);
+
+    }
+
+    @Test
+    public void getExerciseCalories_multipleExercises_returnZero() {
+        UserInfo user = createDefaultUser();
+        int expectedCalories = 0;
+        int dayOfYear = 1;
+        Day emptyDay = new Day(dayOfYear);
+        Calculator calculator = new Calculator(emptyDay);
+        int actualCalories = calculator.getTotalExerciseCalories(user);
+
+        Assert.assertEquals(expectedCalories, actualCalories);
+
+    }
+
+    @Test
+    public void getNetCalories_multipleExercisesMultipleMeals_returnCalories() {
+        int expectedCalories = 5987;
+        UserInfo user = createDefaultUser();
+        Calculator calculator = new Calculator(day);
+        int actualCalories = calculator.getNetCalories(user);
+
+        Assert.assertEquals(expectedCalories, actualCalories);
+
+    }
```

Comment:
As per braicos comments, these test names are too specific and you can have more than one assert per test